presto-hive/src/test/java/com/facebook/presto/hive/TestHivePageSink.java

```diff
@@ -314,7 +314,7 @@ private static ConnectorPageSink createPageSink(HiveTransactionHandle transactio
         return provider.createPageSink(transaction, getSession(config), handle, PageSinkProperties.defaultProperties());
     }
 
-    private static TestingConnectorSession getSession(HiveClientConfig config)
+    protected static TestingConnectorSession getSession(HiveClientConfig config)
```

Comment:
Leave as is. This is not a util class. `TestHivePageSink` is completely for a different purpose. Just copy the following line to the class you want